sympy/core/function.py

```diff
@@ -266,6 +266,71 @@ def nargs(self):
     def __repr__(cls):
         return cls.__name__
 
+    def __pos__(cls):
+        from sympy.operator.operator import Op
```

Comment:
Can we shift this and other similar imports to the top of the file?

Reply:
Negative. It raised cyclic import error, even if I moved it to the not top but bottom of the file.